### ISSUE TITLE
Question: kube-prometheus chart values.yaml

### ISSUE THREAD

**Issue by bamb00** [coreos/prometheus-operator#1343]
**What did you do?**
Does the alertmanager & prometheus "baseImage:, image:" key:value in values.yaml get use in the kube-prometheus chart?
**What did you expect to see?**
baseImage and tag use in the kube-prometheus chart
**What did you see instead? Under which circumstances?**
Cannot find anywhere in the chart those key:value is being use?
**Environment**
prometheus-operator 0.19.0, CentOS 7.4
* Kubernetes version information:

	K8s v1.9.5

* Kubernetes cluster kind: 

	Using kops

* Manifests:

deploy prometheus-operator
deploy kube-prometheus

* Prometheus Operator Logs:
N/A

**Comment by gianrubio** [comment]
@bamb00 yes, you could specify here https://github.com/coreos/prometheus-operator/blob/master/helm/kube-prometheus/values.yaml#L42-L44 and here https://github.com/coreos/prometheus-operator/blob/master/helm/kube-prometheus/values.yaml#L177-L179

**Comment by bamb00** [comment]
Hi @gianrubio,

The question is how's the alertmanager & prometheus (image, repository, tag) in values.yaml being used when helm install kube-prometheus chart?  Is it being use in another way? 

I am a little confused.  Even though it is set in values.yaml but does not get call in kube-prometheus chart.  I review the .yaml in templates and could not find where the image is being call. 

     prometheus:
       image:
         repository: quay.io/prometheus/prometheus
         tag: v2.2.1

     alertmanager:
       image:
         repository: quay.io/prometheus/alertmanager
         tag: v0.14.0


The same image, tag, and repository is set in alertmanager chart and prometheus chart.

Thanks.

**Comment by bamb00** [comment]
@gianrubio Why in kube-prometheus values.yaml and not in the alertmanager & prometheus values.yaml.  Can I override and use the values.yaml from the alertmanager & prometheus chart?

**Comment by gianrubio** [comment]
I can't understand your question, can you elaborate it? Maybe you can paste your values.yaml files and the commands were you ran.

**Comment by bamb00** [comment]
For example the values.yaml in kube-prometheus chart contain the following,

       alertmanager:    
          image:
             repository: repo1_path/alertmanager
             tag: v0.13.1

The values.yaml in alertmanager chart contain the following,
 
         image:
            repository: repo1_path/alertmanager
            tag: v0.14.0

When I execute,
       helm install kube-prometheus --name kube-prometheus --namespace monitoring

The alertmanager image v0.13.1 is deploy and not v0.14.0.  I'm wondering why the image declare in the alertmanager (values.yaml) is not deploy.  This is the same for prometheus chart.
